todo/todo-service/Dockerfile

```diff
@@ -5,6 +5,7 @@ RUN apk add --no-cache curl git openssh
 
 RUN go get -u -v github.com/goadesign/goa/...
 RUN go get -u -v github.com/globalsign/mgo
+RUN go get -u -v github.com/Microkubes/microservice-tools/config
```

Comment:
@anatrajkovska  Install microservice-tools and all dependencies like this:

RUN go get -u -v github.com/Microkubes/microservice-tools/...